resources/views/dinnerform/admin.blade.php

```diff
@@ -0,0 +1,20 @@
+@extends('website.layouts.redesign.generic')
```

Comment:
Is er een reden dat je hier niet kiest voor een wat traditionelere tabel-layout in plaats van de cards die je nu hebt? Is misschien wat overzichtelijker als er meer actief zijn, en zo kun je misschien ook oude forms zichtbaar houden en heractiveerbaar maken?

Reply:
Is inderdaar misschien wel mooier

Reply:
You re-requested a review but didn't do anything with this resolved convesation?